### PR TITLE
Fix Login: Application crashes on logging in from logout page

### PR DESCRIPTION
**Pull request by sumukhah**
on Logout application navigates to '/logout' and it makes api request,then it will show login form.But in
login component, after a successfull login it will redirect to previous screen,In this case '/logout' screen.
because we used componentDidRecieveProp, it will be invoked. this process forms a continuose loop.as the result
application gets crash. This has been resolved by navigating to a different screen '/' after login.

Fixes: #1147

### CODE REVIEW

**Comment by sumukhah** [comment]
This is not my final pull request,
I need some suggestions from you on how to achieve this efficiently.
Thank you

File: src/components/theme/Logout/Logout.jsx [hunk 1]
```diff
@@ -55,7 +55,13 @@ class Logout extends Component {
    * @returns {string} Markup for the component.
    */
   render() {
-    return <Login location={{ query: this.props.location.query }} />;
+    console.log(this.props.location);
```
Comment (nileshgulia1) [inline src/components/theme/Logout/Logout.jsx]: @sumukhah Please remove this console.logs.

Comment (sumukhah) [reply src/components/theme/Logout/Logout.jsx]: Thank you for review @nileshgulia1 ,
will fix it soon,

**Comment by sneridagh** [comment]
@sumukhah Hey, sorry I didn't saw your PR and I did fix it in #1147 as it was an important fix.
Thanks for the contribution anyways! BTW, you need to sign the Plone contribution agreement before you can merge into Volto/Plone. http://docs.plone.org/develop/coredev/docs/contributors_agreement_explained.html

**Pull request closed without merging**